spec/features/user_views_lists_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "User views all lists" do
+  scenario "successfully" do
+    create(:list, title: "Best Sci Fi")
+    book = create(:book, cover_image_url: 'http://placehold.it/140x100')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.